mailpoet/lib/Subscription/Throttling.php

```diff
@@ -34,7 +34,9 @@ public function throttle() {
       if (!empty($subscriberIp)) {
         $subscriptionCount = $this->subscriberIPsRepository->getCountByIPAndCreatedAtAfterTimeInSeconds($subscriberIp, $subscriptionLimitWindow);
         if ($subscriptionCount > 0) {
-          $timeout = $subscriptionLimitBase * pow(2, $subscriptionCount - 1);
+          $timeout = $subscriptionLimitBase * (pow(2, $subscriptionCount - 1));
```

Comment:
Are extra brackets necessary here?

Reply:
> Are extra brackets necessary here?

No, it is a leftover from a local change to test converting with `intval.`
Thanks for noticing, fixed.